lib/elixir/test/elixir/string/chars_test.exs

```diff
@@ -52,6 +52,14 @@ defmodule String.Chars.NumberTest do
   end
 end
 
+defmodule String.Chars.RegexTest do
+  use ExUnit.Case, async: true
+
+  test "regex" do
+    assert to_string(~r/@/) == "@"
```

Comment:
I think the tet should not be that simple. 
It would be good to test for regex with modifiers...
And an empty one  ~r//